### PR TITLE
update home page styles

### LINKED ISSUE #314: Home: update styling and implement top features
Implement the more stable parts of the new home page

#### Designs and Mockups

https://invis.io/DFZBR47WPA7#/436781958_OCW_NexGen_Homepage_11-04-2020_V2

#### Acceptance Criteria:

- [x] Search box (but not the 3 red buttons below)
- [x] Tag line box
  - [x] "Unlocking Knowledge..." 
  - [x] "Learn More" displays "coming soon" pop-up
- [x] New courses (should already be done)
- [x] restyle social buttons
- [x] add corporate logos 
- [x] implement mobile view (https://projects.invisionapp.com/d/main?origin=v7#/console/18552410/437726301/preview?scrollOffset=1705.5)

#### Out of Scope

- Discover Topics
- OCW News
- Learner Stories

### PR DESCRIPTION
**Pull request by aliceriot**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
  - [ ] Tag @ferdi or @pdpinch for review
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested

#### What are the relevant tickets?

part of #314

#### What's this PR do?

this implements the new design for the home template. I didn't see a corresponding mobile design, so I took a stab at a 'column' view to correspond to the 'row' view in the design.

I also removed the React component we were using for the home search page - turns out it's possible to implement the same behavior with just a `<form>` and a pair of `<input>` tags! nice.

#### How should this be manually tested?

check out the mockups and the design, and make sure it looks legit. try out the home page search box too, and make sure that works as it should.

#### Screenshots (if appropriate)
Screenshot from 2020-11-13 12-05-35
Screenshot from 2020-11-13 12-05-27
Screenshot from 2020-11-13 12-05-17
Screenshot from 2020-11-13 12-05-07

### CODE REVIEW

**Comment by abdulkdawson** [comment]
It's looking great nice work.

**Comment by abdulkdawson** [comment]
@aliceriot here is the updated mobile design. [https://projects.invisionapp.com/d/main?origin=v7#/console/18552410/437726301/preview?scrollOffset=1705.5](uhttps://projects.invisionapp.com/d/main?origin=v7#/console/18552410/437726301/preview?scrollOffset=1705.5rl)

**Comment by aliceriot** [comment]
@abdulkdawson great, if it makes sense to you I'll circle back with a second PR to fully implement the mobile design after this merges.

File: src/css/header.scss [hunk 1]
```diff
@@ -81,3 +81,50 @@
     margin-top: -18px;
   }
 }
+
+#new-desktop-header {
+  height: 88px;
+  max-width: 1440px;
```
Comment (noisecapella) [inline src/css/header.scss:87]: Can we use `$max-content-width - 7` for this value?

**Review by noisecapella — CHANGES_REQUESTED** [review]

File: src/css/header.scss (hunk 1 above)
Comment (aliceriot) [reply src/css/header.scss:87]: happy to make the change, but why `- 7`?

File: site/themes/multi_course/layouts/home.html [hunk 2]
```diff
@@ -1,7 +1,73 @@
...
+        <div class="left">
+          <div class="ocw-logo mr-6">
+            <a href="/courses">
+              <img src="/images/ocw_logo_white.png" />
+            </a>
+          </div>
+        </div>
+        <div class="right">
+          <a class="coming-soon btn orange-link py-2 px-3">
+            <div class="text-white font-weight-bold w-100 coming-soon">Give now</div>
```
[the remark this answers is not in the document]
Comment (aliceriot) [reply site/themes/multi_course/layouts/home.html:28]: oh haha

File: src/css/header.scss (hunk 1 above)
Comment (noisecapella) [reply src/css/header.scss:87]: Just because it looks like the variable `$max-content-width` is `1447`

**Review by noisecapella — APPROVED** [review]

File: src/css/header.scss (hunk 1 above)
Comment (aliceriot) [reply src/css/header.scss:87]: ah I think I meant to use `$max-content-width` and just forgot to change it

**Pull request merged**